ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-ref.metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+	"path": "optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-ref.yaml",
+	"owners": {
+		"approvers": [
+			"dptp"
```

Comment:
Nice try ;)

This goes to `OWNERS` file (ci-operator/step-registry/optional-operators/ci/run-bundle/OWNERS) (and the metadata is generated from it).

Second, DPTP will not own this step. It was our understanding that the SDK team would maintain this step for their users (operator authors): this is a principle used across the whole OpenShift CI. For example, OCP installer team maintans the steps that iinstall OCP and are used across the org.